### PR TITLE
[CARBONDATA -816] Added Example For HIve Integration

### PR DESCRIPTION
**Pull request by anubhav100**
Added Example For HIve Integration

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1467/

File: format/src/main/thrift/sort_index.thrift [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
 
 /**
- * File format description for carbon sort index file
+ * File format description for CarbonData sort index file
```
Comment (chenliang613) [inline format/src/main/thrift/sort_index.thrift]: why change thrift files?

Comment (anubhav100) [inline format/src/main/thrift/sort_index.thrift]: Changes come from master branch should I remove these extra commits?

Comment (anubhav100) [inline format/src/main/thrift/sort_index.thrift]: I created this branch from master and then rebased with hive branch that's why there are some changes come from master branch

Comment (chenliang613) [inline format/src/main/thrift/sort_index.thrift]: Suggest creating a new PR based on apache/hive

**Comment by chenliang613** [comment]
Please put all files under integration/hive for all hive related PR

**Pull request closed without merging**